snippets/desktop-guide/wpf/styles-and-templates-intro/csharp/Photo.cs

```diff
@@ -1,8 +1,9 @@
-// // Copyright (c) Microsoft. All rights reserved.
+﻿// // Copyright (c) Microsoft. All rights reserved.
 // // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 namespace IntroToStylingAndTemplating
 {
+    // <PhotoClass>
```

Comment:
Snippets don't need to start with `Snippet`? I just learned something.

Reply:
At least according to the spec and stuff for docfx. It just says "NAME" as the value. I think Snippet may be in our blood just because of the history with parsnip which did require it start with Snippet.